File: src/Task2.java
import java.util.Scanner;

public class Task2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double a, b, c, d, u;
        System.out.println("Введіть межі для x");
        a = scanner.nextDouble();
        b = scanner.nextDouble();
        double x = a, hx = (b - a) / 7;
        System.out.println("Введіть межі для y");
        c = scanner.nextDouble();
        d = scanner.nextDouble();
        double y = c, hy = (d - c) / 7;
        System.out.print("y\\x ");
        for (int i = 0; i < 8; i++)
            System.out.printf("%8.2f", x + hx * i);
        System.out.println();
        for (int i = 0; i < 8; i++) {
            System.out.printf("%5.2f", y);
            x = a;
            for (int j = 0; j < 8; j++) {
                u = Math.pow(Math.sin(x), 2) + Math.cos(y);
                System.out.printf("%8.2f", u);
                x += hx;
            }
            System.out.println();
            y += hy;
        }
    }
}

File: src/Task_2.kt
import java.util.*
import kotlin.math.cos
import kotlin.math.sin
import kotlin.math.pow

object Task_2 {
    @JvmStatic
    fun main(args: Array<String>) {
        val scanner = Scanner(System.`in`)
        var u: Double
        println("Введіть межі для x")
        val a = scanner.nextDouble()
        val b = scanner.nextDouble()
        var x = a
        val hx = (b - a) / 7
        println("Введіть межі для y")
        val c = scanner.nextDouble()
        val d = scanner.nextDouble()
        var y = c
        val hy = (d - c) / 7
        print("y\\x ")
        for (i in 0..7) System.out.printf("%8.2f", x + hx * i)
        println()
        for (i in 0..7) {
            System.out.printf("%5.2f", y)
            x = a
            for (j in 0..7) {
                u = sin(x).pow(2.0) + cos(y)
                System.out.printf("%8.2f", u)
                x += hx
            }
            println()
            y += hy
        }
    }
}
